lib/presentation/pages/profile_screen.dart

```diff
@@ -0,0 +1,349 @@
+import 'package:flutter/material.dart';
+import 'package:nowingoogle/data/utils/capitalize.dart';
+import 'package:nowingoogle/data/utils/get_avatar.dart';
+import 'package:nowingoogle/domain/entities/speaker.dart';
+import 'package:nowingoogle/domain/entities/user.dart';
+import 'package:nowingoogle/domain/entities/volunteer.dart';
+import 'package:nowingoogle/domain/enums/user_roles.dart';
+import 'package:nowingoogle/presentation/pages/layouts/io_footer.dart';
+import 'package:nowingoogle/presentation/pages/layouts/profile/profile_header.dart';
+import 'package:nowingoogle/presentation/pages/layouts/profile/profile_section.dart';
+import 'package:nowingoogle/presentation/pages/widgets/title_subtitle_listview.dart';
+import 'package:nowingoogle/presentation/pages/widgets/primary_button.dart';
+import 'package:nowingoogle/presentation/pages/widgets/secondary_button.dart';
+
+class ProfileScreen extends StatefulWidget {
+  // final User user;
+  const ProfileScreen({super.key});
+
+  @override
+  State<ProfileScreen> createState() => _ProfileScreenState();
+}
+
+class _ProfileScreenState extends State<ProfileScreen> {
+  var isEditing = false;
+  @override
+  Widget build(BuildContext context) {
+    final User user = ModalRoute.of(context)?.settings.arguments as User;
```

Comment:
Please move all of these outside of build method and use appropriate lifecycle methods